sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -3305,7 +3311,41 @@ SIREPO.app.directive('jobsList', function(requestSender, appState, $location, $s
             appState.clearModels(appState.clone(SIREPO.appDefaultSimulationValues));
             $scope.getJobs();
 
+            panelState.waitForUI(() => {
+                $('#' + panelState.modalId('jobsListModal')).on('shown.bs.modal', function() {
+                $scope.getJobs();
+                });
+            });
+
```

Comment:
Remove this blank line and the one 2 lines down.

Reply:
3f4d10d86518e0e428966ed7697717e1d82d6e96